src/client/Microsoft.Identity.Client/Platforms/Android/AndroidHttpClientFactory.cs

```diff
@@ -17,7 +17,7 @@ public HttpClient GetHttpClient()
 
             var httpClient = new HttpClient(
                 // As per Xamarin guidance https://docs.microsoft.com/en-us/xamarin/android/app-fundamentals/http-stack?tabs=windows
-                new Xamarin.Android.Net.AndroidClientHandler());
+                new Xamarin.Android.Net.AndroidMessageHandler());
```

Comment:
Ok, looks like `AndroidClientHandler` is obsolete.

Reply:
Yes